controllers.err.js

```diff
@@ -1,3 +1,7 @@
 exports.notFound = (req, res, next) => {
-    res.status(404).send({msg : 'Path not found'});
-}
\ No newline at end of file
+    res.status(404).send({msg : 'Not Found'});
+}
+
+exports.handleCustomError = (err, req, res, next) => {
+    res.status(400).send({ msg: "Bad Request" });
+  };
```

Comment:
(again from my comment on app.js) always good to have a 500 error handler at the very bottom to catch everything else. Take a look at the '4. Refactoring to Individual Error Handling Blocks' section of the error handling notes (https://notes.northcoders.com/courses/js-back-end/error-handling) again to see the pattern I want you to try and build out